src/invoices/Voucher/VoucherViewLayer.js

```diff
@@ -0,0 +1,140 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import ReactRouterPropTypes from 'react-router-prop-types';
+import { FormattedMessage } from 'react-intl';
+import { get } from 'lodash';
+
+import {
+  Button,
+  Col,
+  Icon,
+  Layer,
+  MenuSection,
+  Pane,
+  PaneMenu,
+  Row,
+} from '@folio/stripes/components';
+import {
+  AppIcon,
+  stripesConnect,
+} from '@folio/stripes/core';
+
+import { LoadingPane } from '../../common/components';
+import {
+  invoiceResource,
+  VOUCHER_BY_ID,
+  VOUCHER_LINES,
+} from '../../common/resources';
+import {
+  expandAll,
+  toggleSection,
+} from '../../common/utils';
+import VoucherView from './VoucherView';
+
+class VoucherViewLayer extends Component {
```

Comment:
why not to make this functional as well?

Reply:
Updated